scripts/README.md

```diff
@@ -0,0 +1,3 @@
+# Development Scripts
```

Comment:
maybe its better to name the folder "psng-dev-scripts". "scripts" maybe is used by many users.

Reply:
I'm not sure we should try to avoid conflicts like that? If people clone this repo straight into their config folder, there are likely to be many other issues over time than just a folder name matching :)

Reply:
i think its important to keep the update progress very easy for the users. If they can to a simple git pull instead of copying dirs to the right place (this was the intention to do the full files-cleanup and rename).
Maybe you can place the scripts-dir as subdir in psng-folder?

Reply:
In my experience, doing that won't work reliably for people - git will absolutely cause problems for people over time :)

The symlink method (for the 3 folders we expect people to use) would be far more reliable, and means users don't have to deal with or understand git.

Reply:
OK, if you have the experience. I don´t have that.
But then i think we should update the install-manual to do that by git-clone in a separate dir + linking to config.

Reply:
Looking at the readme/install docs - they actually don't need a change yet (this PR won't affect them). I'll open another issue re creating a simple install/update command. I think we can make it very simple for users by just adding a script to handle the install/update :)

Reply:
Nice!